lib/readwise/client.rb

```diff
@@ -48,7 +49,12 @@ def export_page(page_cursor: nil, updated_after: nil, book_ids: [])
           readwise_url: item['readwise_url'],
           source: item['source'],
           source_url: item['source_url'],
-          tags: item['book_tags'],
+          tags: item['book_tags'].map do |tag|
+            Tag.new(
```

Comment:
It might be good to move this logic into `Tag`, so here we would just call something like `Tag.for(tag)`.

Reply:
(which may mean turning `Tag` into a proper class than a struct)

Reply:
Would that mean doing the same for a highlight as well, or is there something that distinguishes the two cases here where tag would need to be a class and highlight doesn't?

Reply:
Probably at some point, so that `client.rb` doesn't become too complex. But currently there's only one use of `Highlight.new` and two for `Tag.new` so I feel it's better to avoid that duplication.

Reply:
I see what you're saying, makes sense! Do we want to create a new ticket that focuses on turning `Highlight` and `Tag` into a formalized class and call this current ticket done, or do we want to go ahead and turn this `Tag` struct into a class now, and deal with the `Highlight` struct later? I'm happy to do either 👍

Reply:
I'd keep them as structs for this PR and we can make them all classes in a single sweep/PR.

Reply:
Sounds good, went ahead with the id renaming to `tag_id`, so I believe that covers everything.